README.rst

```diff
@@ -2,17 +2,16 @@ intel-iot-refkit
 ################
 
 This README file contains information on setting up, building, and booting
-Iot Reference OS Kit for Intel(r) Architecture project.
+IoT Reference OS Kit for Intel(r) Architecture project.
 
```

Comment:
Yes the "the" should be there. The comma before the and is a standard "oxford comma" and helps clarify the list of items.